src/CognitiveServices/CognitiveServices.Test/ScenarioTests/CognitiveServicesAccountTests.ps1

```diff
@@ -65,23 +65,16 @@ function Test-NewAzureRmAllKindsOfCognitiveServicesAccounts
 		New-AzResourceGroup -Name $rgname -Location 'West US';
 		
 		# Create all known kinds of Cognitive Services accounts.
-		Test-CreateCognitiveServicesAccount $rgname 'AcademicTest' 'Academic' 'S0' $locWU
```

Comment:
Why are you removing all of these test cases?

Reply:
These APIs are deprecated and are no longer supported so re-running recording would fail.

Reply:
When were these APIs deprecated? Is the move from the old SDK version to the new SDK version causing these to no longer be supported, or would the creation scenarios fail in the current version of Cognitive Services? This seems like a breaking change to me.

Reply:
These are deprecated on the server side. From control-plane perspective there is no breaking change since there is no change in control-plane contract. The APIs are entirely data-plane.

Reply:
Would the above test case work in the old version of the API, or was this deprecated in every version of the API? Basically, would a customer be able to run this successfully in the old API version and not the new API version?

Reply:
Customer would not be able to run this successfully using any version of PS/CLI/SDK because the resource kinds have been deprecated. Each "API" is just a resource kind.